src/App.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import Header from './components/Header/Header';
+import Front from './pages/Front/Front';
+import {Route} from 'react-router-dom';
+import Login from './pages/Login/Login';
+import StudentTracker from './pages/StudentTracker/StudentTracker';
+
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
```

Comment:
Would be better if we move the routes into  a new folder in src folder and name it routes